js/index.js

```diff
@@ -0,0 +1,36 @@
+const addToInputList = document.querySelector("#addName");
+const addButton = document.querySelector("#addNameToList");
+const addToOutputList = document.querySelector("#outputList");
+
+// Function to append 
+function addToList() {
+    const ol = addToOutputList.querySelector("ol");
+    const li = document.createElement("li");
+    const paragraph = document.createElement("p");
+
+    li.innerHTML = addToInputList.value;
+
+    if (!li.innerHTML) {
+        console.warn('Input text field cannot be empty');
+    }
+    else {
+        ol.appendChild(li);
+        ol.appendChild(paragraph);
+        addToInputList.value = "";
+    }
+};
+
+
+// Execute when the user click on ADD button on Interface
+addButton.addEventListener("click", () => {
+        addToList();
```

Comment:
I like that you added the logic to add item to the list in a separate function and called it here. Good job 👍